.eslintrc.json

```diff
@@ -55,7 +55,9 @@
     "@typescript-eslint/require-array-sort-compare": "error",
     "@typescript-eslint/restrict-plus-operands": "error",
     "@typescript-eslint/type-annotation-spacing": "error",
-    "@typescript-eslint/unbound-method": "error"
+    "@typescript-eslint/unbound-method": "error",
```

Comment:
Do you recall which lines are triggering this issue. Would prefer either fix the error, or at an ignore on the line. Not global.

Reply:
The issue was with adding enums causing an error `already declared in the upper scope`
Adding those two lines was as suggested here:
https://github.com/typescript-eslint/typescript-eslint/issues/2466